x/superfluidstaking/README.md

```diff
@@ -0,0 +1,77 @@
+# Superfluid staking
+
+Superfluid staking is to use the OSMO tokens put on liquidity pools to be used both for swap & staking.
+
+## Potential problems
+
+### Is it secure to use it both for staking and swap?
+
+In current design, liquidity is not used enough and therefore, using it both for swap and staking won't be a problem.
+As we update the pools to use liquidity more efficiently, almost all of those liquidity could be used for swap.
+In that case, the amount of OSMO in the pool does not participate in increasing the security of the chain.
+
+### How to determine the amount of OSMO delegation from the pool to validators?
+
+Time Average Osmo Amount per LP token * Risk Rating of LP token
+
+### On slash event, OSMO will be burnt from the pool directly or LP tokens will be burnt?
+
+I think burning LP token would be nicer and pretty easier.
+LP token's value could be calculated based on how much OSMO is in it and also calculate burn amount from that.
+
+If burn OSMO, it will cause a problem of price change.
```

Comment:
I think we should record at each epoch how many osmo each LP share is being treated as. Then at time of slash, you calculate the current osmo worth of each LP share. You then burn `slashing_rate * osmo_per_LP_share_at_time_of_slash / current_osmo_per_LP_share` number of LP shares.